dev/benchmarks/complex_layout/lib/main.dart

```diff
@@ -0,0 +1,559 @@
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          children: <Widget>[
+            new Stack(
+              children: <Widget>[
+                new SizedBox(
+                  height: 230.0,
+                  child: new AssetImage(
+                    name: "packages/flutter_gallery_assets/top_10_australian_beaches.png"
+                  )
+                ),
+                new Theme(
+                  data: new ThemeData.dark(),
+                  child: new Row(
+                    mainAxisAlignment: MainAxisAlignment.end,
+                    children: <Widget>[
+                      new IconButton(icon: Icons.edit, onPressed: () { print("Pressed edit button"); }),
+                      new IconButton(icon: Icons.zoom_in, onPressed: () { print("Pressed zoom button"); })
+                    ]
+                  )
+                ),
+                new Positioned(
+                  bottom: 4.0,
+                  left: 4.0,
+                  child: new Container(
+                    decoration: new BoxDecoration(
+                      backgroundColor: Colors.black54,
+                      borderRadius: 2.0
+                    ),
+                    padding: new EdgeInsets.all(4.0),
+                    child: new RichText(
+                      text: new TextSpan(
+                        style: new TextStyle(color: Colors.white),
+                        children: <TextSpan>[
+                          new TextSpan(
+                            text: "Photo by "
+                          ),
+                          new TextSpan(
+                            style: new TextStyle(fontWeight: FontWeight.bold),
+                            text: "Magic Mike"
+                          )
+                        ]
+                      )
+                    )
+                  )
+                )
+              ]
+            )
+            ,
+            new Padding(
+              padding: new EdgeInsets.all(8.0),
+              child: new Column(
+                crossAxisAlignment: CrossAxisAlignment.stretch,
+                children: <Widget>[
+                  new Text("Where can you find that amazing sunset?", style: Theme.of(context).textTheme.body2),
+                  new Text("The sun sets over stinson beach", style: Theme.of(context).textTheme.body1),
+                  new Text("flutter.io/amazingsunsets", style: Theme.of(context).textTheme.caption)
+                ]
+              )
+            )
+          ]
+        )
+      )
+    );
+  }
+}
+
+class ItemGalleryBox extends StatelessWidget {
+  ItemGalleryBox(this.index);
+
+  final int index;
+
+  @override
+  Widget build(BuildContext context) {
+    List<String>tabNames = <String>[
+      "A", "B", "C", "D"
+    ];
+
+    return new SizedBox(
+      height: 200.0,
+      child: new TabBarSelection<String>(
+        values: tabNames,
+        child: new Builder(
+          builder: (BuildContext context) {
```

Comment:
Copy pasted the code from the examples, will see if it works without the builder too